projects/UWMadison_Kwan.yaml

```diff
@@ -0,0 +1,5 @@
+Department: Pharmacy
+Description: ''
```

Comment:
Hi @cjabplanalp, is there a good description that you can provide for this project?